src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -7,4 +7,5 @@ CA1069 | Design | Info | EnumShouldNotHaveDuplicatedValues, [Documentation](http
 CA2011 | Reliability | Info | AvoidInfiniteRecursion, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2011)
 CA2012 | Reliability | Hidden | UseValueTasksCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2012)
 CA2013 | Reliability | Warning | DoNotUseReferenceEqualsWithValueTypesAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2013)
-CA2014 | Reliability | Warning | DoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2014)
\ No newline at end of file
+CA2014 | Reliability | Warning | DoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2014)
+CA2015 | Reliability | Warning | DoNotDefineFinalizersToTypesDerivedFromMemoryManager, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2015)
```

Comment:
```suggestion
CA2015 | Reliability | Warning | DoNotDefineFinalizersForTypesDerivedFromMemoryManager, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2015)
```

Reply:
This entry is auto-generated by the code fix based on the type name used in the analyzer implementation. You may want to also update the analyzer type name here.

Reply:
> You may want to also update the analyzer type name here.

Analyzer name changed, forgot this